web/src/features/landingPage/components/Sidebar.tsx

```diff
@@ -235,8 +236,8 @@ export const Sidebar: React.FunctionComponent = () => {
         {renderLargeCollaborate()}
         <Subheading title="Support" />
         <Content>
-          <Typography>To report bugs or receive support on technical issues, please email:</Typography>
-          <Email href={'mailto:BCWS.PredictiveServices@gov.bc.ca'}>BCWS.PredictiveServices@gov.bc.ca</Email>
+          <Typography>To report bugs or receive support on technical issues, please click below:</Typography>
+          <FeedbackButton color="primary" />
```

Comment:
I think we need a little white space above the button.

nit: What do you think about centering the button horizontally?